theNumber.js

```diff
@@ -0,0 +1,108 @@
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+let guessCount = 0 //turn counter for guess
+
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+// ********* Game begins below this line*************
+
+const min = 1;
+const max = 100;
+const acceptableNumbers = aceptNumb();
+
+let thePlayerNumber;
+//stores computer guesses in an array
+let alreadyGuessed = [];
+let compGuesses = comparingNumbers()
+// this function generates the computer's guesses
+function compGuesses(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```

Comment:
When running the code, it breaks and throws an error due to a naming conflict in these lines. In order to fix this you can just change the name of either the variable or the function so that they are not both 'compGuesses'